README.md

```diff
@@ -1,8 +1,65 @@
+- High availability with clustering
+- Logging
+- Documented backup procedures
+- Proven performance metrics
+- Support for hybrid cloud offerings
+
+From a partner perspective, both technology and integrator, the solution must meet these additional software requirements.
+
+- Language agnostic
+- Simple, well documented deployment options
+- CLI integration for development testing
+- Support for unit testing
+- Ability to integrate with existing CI solutions
+- Integrated workflow automation support
+- Offline development support
+- Industry support for methodology and technology
+- Token storage and retrieval
+- Ability to reuse functionality
+
+
+# Current state of integrations with GitHub Enterprise
+Most integrators started developing with GitHub using our [REST framework referenced under API v3](https://developer.github.com/v3/).  This version of the API would allow integrations to act on behalf of a GitHub user using either a [PAT (personal access token)](https://developer.github.com/v3/auth/#via-oauth-tokens) or with the [OAuth Authorizations API](https://developer.github.com/v3/oauth_authorizations/#create-a-new-authorization).  The integrations that were built using these techniques were normally point to point solutions and very heavyweight.  A developer would need to build out a full stack web application to leverage the OAuth Authorization workflow.
+
+![](images/Current.png)
+
+Therefore, the only people building integrations for GitHub were larger ISVs.  These integrations were considered very heavyweight.  It required them to build a webhook receiver to accept JSON payloads from GitHub.  They would need to parse the JSON payload and perform some action based on the contents of that JSON payload and respond to GitHub using the GitHub API.  To build a very simple integration, like a bot, it could take days to build and deploy.  For ISVs these integrations could take months.  
+
+Due to the security concerns of OAuth, GitHub moved to a new model for extending it's platform called [GitHub Apps](https://developer.github.com/apps/building-github-apps/).  A GitHub developer, @bkeepers, built a reference implementation of a GitHub App in Node.js called [Probot](https://github.com/probot/probot).  Probot is great example of how you could extend GitHub :sparkles:.  This still uses the methodology where GitHub events are triggered via Webhooks and Probot is running an [Express](https://expressjs.com/) web server waiting to accept the JSON payload and perform some action based on that data.  Since we don't know when an action will get triggered, the Probot App must be always available.  This creates some concern with our Enterprise customers as some regulated workflows will depend on this integration for moving code throughout the SDLC.  If this capability is down, an exception would need to be established in InfoSec to allow a manual trigger to move the state forward.  
+
+# How are **Ghapplets** (GitHub Applets) different?
```

Comment:
This is a little pedantic, but as we usually capitalize the `H` in GitHub, could these be `GHApplets`?

Reply:
yeah, i've thought about it both ways.  Wanted to see how it looks in written form.  I'll capitalize it for now to see how that reads.